ex1/main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import "fmt"
+
+
+
+func main() {
+    
+    
+var OS int
+var Ayo string
+var learner bool
```

Comment:
Your Vars should be defined with small letters starting their names inside the `main` function. You don't need to capitalise their first letter.